### PR TITLE
Not requiring a user name for adding jobs via REST API

### PR DESCRIPTION
**Pull request by paulsaxe**
Since the user must be logged in, removed the requirement to have a user name when using the REST API to submit a job. If the user name is not given it is assumed to be that of the user logged in.

### CODE REVIEW

**Comment by janash** [comment]
Is this ready to be reviewed?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```
Comment (janash) [inline LICENSE:1]: Isn't there a license on main already? I'm confused as to why this is showing up as a new file in the PR.

**Review by janash — APPROVED** [review]
lgtm 👍🏻

File: LICENSE (hunk 1 above)
Comment (paulsaxe) [reply LICENSE:1]: I think this is the same license ... I copied it locally to be able to upload to Conda Forge, which requires a license file. I think nothing will change when committing this.

**Pull request merged**